languagetool-core/src/main/java/org/languagetool/rules/patterns/PatternToken.java

```diff
@@ -434,7 +434,7 @@ public int getSkipNext() {
    * The minimum number of times the element needs to occur.
    */
   public int getMinOccurrence() {
-    return minOccurrence;
+    return mayBeOmitted ? 0 : 1;
```

Comment:
This may be rare, but it should be possible to have a min occurrence of e.g. 2.

Reply:
`setMinOccurrence` forbids it now

Reply:
Yes, but should we do so? There's `min="x"` in the XML and I think this should allow values > 1, even if this is rare.

Reply:
I don't know, I only took existing contracts and optimized based on them :)

I think, in the unlikely case where `min="2"` is really-really needed, it can be simulated by creating two similar `<token>` tags.

Reply:
Ah sorry, I missed the fact that the restriction to 0 or 1 existed forever...